build-test.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
+# For scenario testing purposes
 
-./build.sh
+if [[ ! -f .env ]]; then
+    echo "Default build arguments don't exists. Creating one from default value."
+    cp .example.env .env
+fi
 
-docker build -t kartoza/postgis:manual-build -f Dockerfile.test .
+docker-compose -f docker-compose.build.yml build postgis-test
```

Comment:
Simplify build command. Now the tags are declared in the docker-compose file. The stage are also declared in the docker-compose file. We only need to specify which service to build.